src/main/java/org/broadinstitute/hellbender/tools/exome/TargetTableAnnotationManager.java

```diff
@@ -0,0 +1,101 @@
+package org.broadinstitute.hellbender.tools.exome;
+
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+
+import java.util.*;
+import java.util.function.Function;
+
+/**
+ * Target table annotation collection.
+ * <p>
+ *     Holds information on what annotations are present in the target table file, what columns contain them and how to
+ *     construct {@link TargetAnnotationCollection} instances to query these per target.
+ * </p>
+ *
+ * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ */
+final class TargetTableAnnotationManager {
+
+    private Set<TargetAnnotation> annotationSet;
```

Comment:
However here is still better to just say "Set" as there is no need to indicate the actual implenetation used in the member declaration.